packages/augur-ui/src/modules/account/selectors/has-stake-in-market.ts

```diff
@@ -0,0 +1,32 @@
+import { createSelector } from 'reselect';
+import { Getters } from '@augurproject/sdk/build';
```

Comment:
are we supposed to be referencing the build folder?

Reply:
good catch, nope. my mistake

Reply:
fixed